index.js

```diff
@@ -4,27 +4,23 @@ module.exports = function (img, selection_canvas, onCrop) {
   if('function' === typeof selection_canvas)
```

Comment:
Does it make sense to drop `bundle.js` from the repo? It's unpopulated and is build output, which (I, at least) generally leave out of git.

(Left this comment here because Github doesn't seem to let me comment on the `bundle.js` file directly.)